crypto/ffc/ffc_params_generate.c

```diff
@@ -77,12 +77,12 @@ static int ffc_validate_LN(size_t L, size_t N, int type, int verify)
         ERR_raise(ERR_LIB_DH, DH_R_BAD_FFC_PARAMETERS);
 # endif
     } else if (type == FFC_PARAM_TYPE_DSA) {
-        if (L == 1024 && N == 160)
-            return 80;
-        if (L == 2048 && (N == 224 || N == 256))
-            return 112;
-        if (L == 3072 && N == 256)
+        if (L >= 3072 && N >= 256)
```

Comment:
These changes also break fips...

Reply:
The buildinfo is already correct for this one if #ifndef FIPS_MODULE is used.

Reply:
This doesn't bread FIPS, it's already in the `#else` part of an `#ifdef FIPS_MODULE` chunk.

Reply:
you are right..